src/plugins/event-plugins/XhrPlugin.ts

```diff
@@ -207,19 +212,25 @@ export class XhrPlugin extends MonkeyPatched<XMLHttpRequest, 'send' | 'open'> {
     private handleXhrAbortEvent = (e: Event) => {
         const xhr: XMLHttpRequest = e.target as XMLHttpRequest;
         const xhrDetails = this.xhrMap.get(xhr);
-        const errorName = 'XMLHttpRequest abort';
-        this.handleXhrDetailsOnError(xhrDetails, errorName);
+        if (xhrDetails) {
+            this.handleXhrDetailsOnError(
+                xhrDetails,
+                xhr,
+                'XMLHttpRequest abort'
+            );
+        }
     };
 
     private handleXhrTimeoutEvent = (e: Event) => {
         const xhr: XMLHttpRequest = e.target as XMLHttpRequest;
```

Comment:
agreed

Reply:
This is out of scope for this change -- it is unrelated to the bug fix.

"XHR" is an abbreviation of "XML HTTP Request", so "xhrRequest" is redundant.

Reply:
These renamings were to avoid repeating Xhr so many times, which is implied in plugin named XhrPlugin. I've created a separate PR #455 to restore these changes and remove the public getter, as you requested